<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdCrawler</class>
 <widget class="QMainWindow" name="AdCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdCrawler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_url">
             <property name="text">
              <string>사이트 URL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_url"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_cnt">
             <property name="text">
              <string>반복횟수</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_cnt">
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="grp_device">
           <property name="title">
            <string>디바이스 종류</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radio_both">
              <property name="text">
               <string>PC, Mobile</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_pc">
              <property name="text">
               <string>PC</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_mobile">
              <property name="text">
               <string>Mobile</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>작업 시작</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_bar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="log_browser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
